telco-hub/configuration/reference-crs/required/acm/acmProvisioning.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: metal3.io/v1alpha1
+kind: Provisioning
+metadata:
+  name: provisioning-configuration
+spec:
+  watchAllNamespaces: true
+
```

Comment:
For some bmc the way the boot ISO is configured doesn't work and you need to add this to the spec. We should put this in but leave it commented out for now.
```
disableVirtualMediaTLS: true
```